index.html

```diff
@@ -37,7 +37,51 @@
         </div>
     </nav>
 
+    <nav class="side-navbar relative">
+        <p> <a href="https://stackoverflow.com/"> Home </a> </p>
+        <p class="mb-0"> Public </p>
+        <ul>
+            <li class="active">
```

Comment:
This class is too generic. Active could be 100 things. Let's rename to something like `sidebar-item--active'.